jetbrains-core/src/software/aws/toolkits/jetbrains/services/telemetry/DefaultTelemetryPublisher.kt

```diff
@@ -38,16 +38,18 @@ class DefaultTelemetryPublisher(
         }
     }
 
-    override fun sendFeedback(sentiment: Sentiment, comment: String) {
-        client.postFeedback {
-            it.awsProduct(clientMetadata.productName)
-            it.awsProductVersion(clientMetadata.productVersion)
-            it.os(clientMetadata.os)
-            it.osVersion(clientMetadata.osVersion)
-            it.parentProduct(clientMetadata.parentProduct)
-            it.parentProductVersion(clientMetadata.parentProductVersion)
-            it.sentiment(sentiment)
-            it.comment(comment)
+    override suspend fun sendFeedback(sentiment: Sentiment, comment: String) {
```

Comment:
Style question, if the function is all `withContext` wouldn't it make more sense to do 
`fun.... (...) = withContext(context) {
}

Reply:
sure

Reply:
in this case we're not using the response so i'm leaving as is